src/apiCalls.js

```diff
@@ -0,0 +1,23 @@
+
+
+const getMovies = async(type, id) => {
+  let url = 'https://rancid-tomatillos.herokuapp.com/api/v2/movies'
+
+  if(id) {
+    url = `https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`
+  } 
+  if (type === 'videos') {
+    url = `https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}/${type}`
+  }
+  
```

Comment:
I love that this fetch is reusable, but I'm still seeing a lot of repeated text here. Maybe we could refactor to send the entire path through as an argument. Then we could skip the conditional and the repeated endpoints!